debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CassandraConnectorTask.java

```diff
@@ -129,15 +130,22 @@ private void initHttpServer() {
         contextHandler.addServlet(new ServletHolder(new HealthCheckServlet(registerHealthCheck())), "/health");
     }
 
-    private void initProcessorGroup() throws IOException {
-        processorGroup = new ProcessorGroup("Cassandra Connector Task");
-        processorGroup.addProcessor(new SchemaProcessor(taskContext));
-        processorGroup.addProcessor(new CommitLogProcessor(taskContext));
-        processorGroup.addProcessor(new SnapshotProcessor(taskContext));
-        processorGroup.addProcessor(new QueueProcessor(taskContext));
-        if (taskContext.getCassandraConnectorConfig().postProcessEnabled()) {
-            processorGroup.addProcessor(new CommitLogPostProcessor(taskContext));
+    private void initProcessorGroup() throws Exception {
+        try {
+            processorGroup = new ProcessorGroup();
+            processorGroup.addProcessor(new SchemaProcessor(taskContext));
+            processorGroup.addProcessor(new CommitLogProcessor(taskContext));
+            processorGroup.addProcessor(new SnapshotProcessor(taskContext));
+            processorGroup.addProcessor(new QueueProcessor(taskContext));
+            if (taskContext.getCassandraConnectorConfig().postProcessEnabled()) {
+                processorGroup.addProcessor(new CommitLogPostProcessor(taskContext));
+            }
         }
+        catch (Exception e) {
+            LOGGER.error("Failed to initiate Processor Group.", e);
```

Comment:
Let's either log _or_ rethrow. Otherwise, the exception would likely end up in the logs twice.